app/pages/project/images/ImagesPage.tsx

```diff
@@ -98,6 +100,12 @@ export function ImagesPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Images24Icon />}>Images</PageTitle>
+        <DocsPopover
+          heading="Images"
+          icon={<Images16Icon />}
+          summary="Images are templates of virtual machines that can be used to create new instances."
```

Comment:
They are templates of disks, not instances, and it's not exactly a template in that for me a template is like a form with slots you can fill in. "Snapshot" is a better word, but that is its own thing. I don't have a good suggestion here yet.

Reply:
Claude suggests "point-in-time copy", that's kind of interesting, though we might reserve that for snapshots.

Reply:
Maybe `blueprints`?
"Images serve as blueprints of disks, used to create new virtual machine instances."

Reply:
Maybe we avoid it by saying what they do without saying what they are.

> Images let you create a new disk based on an existing one. Images can be uploaded directly or created from a snapshot.

The “you” feels a little too casual. I hope most people using the system should have some sense of what an image is.